src/arrayMethodJoin.js

```diff
@@ -4,9 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
```

Comment:
Nice job!! =)
better change variable name to something more intuitive .

Reply:
Agree with @nikolay-li, result is bad name